repos/fdbt-site/src/pages/viewTimeRestrictions.tsx

```diff
@@ -42,33 +44,69 @@ const formatDayRestriction = (timeRestriction: PremadeTimeRestriction, day: stri
     );
 };
 
-const ViewTimeRestrictions = ({ timeRestrictions, referer }: TimeRestrictionProps): ReactElement => (
-    <BaseLayout title={title} description={description} showNavigation referer={referer}>
-        <div className="govuk-grid-row">
-            <div className="govuk-grid-column-one-third">
-                <SubNavigation />
-            </div>
+const ViewTimeRestrictions = ({ timeRestrictions, referer, csrfToken }: TimeRestrictionProps): ReactElement => {
+    const [popUpState, setPopUpState] = useState({
```

Comment:
As discussed let's initialise this to undefined and get rid of isVisible

Reply:
Good shout! That is now done. Please have a look and if happy mark as resolved.